server/src/main/java/dev/slimevr/tracking/trackers/TrackerPosition.java

```diff
@@ -1,148 +0,0 @@
-	}
-
-	TrackerPosition(String designation, Optional<TrackerRole> trackerRole, int bodyPart) {
-		this.designation = designation;
-		this.trackerRole = trackerRole;
-		this.bodyPart = bodyPart;
-	}
-
-	/** Indexed by `BodyPart` int value. EFFICIENCY FTW */
-	private static final TrackerPosition[] byBodyPart;
-	static {
-		// Determine maximum value of BodyPart. Kinda hacky, but this will
-		// prevent breakage if the max value changes.
-		int max = 0;
-		for (var field : BodyPart.class.getFields()) {
-			if (!Modifier.isStatic(field.getModifiers()) || !field.getType().equals(int.class)) {
-				continue;
-			}
-			try {
-				var v = field.getInt(null);
-				max = Math.max(max, v);
-			} catch (IllegalAccessException e) {
-				// unreachable
-				java.util.logging.Logger.getGlobal().log(Level.SEVERE, "Reached unreachable code");
-			}
-		}
-		byBodyPart = new TrackerPosition[max + 1];
-
-		for (var tp : TrackerPosition.values) {
-			byBodyPart[tp.bodyPart] = tp;
-		}
-	}
-
-	private static final Map<String, TrackerPosition> byDesignation = new HashMap<>();
-	private static final EnumMap<TrackerRole, TrackerPosition> byTrackerRole = new EnumMap<>(
-		TrackerRole.class
-	);
-	static {
-		for (TrackerPosition tp : values()) {
-			byDesignation.put(tp.designation.toLowerCase(), tp);
-			tp.trackerRole.ifPresent((tr) -> byTrackerRole.put(tr, tp));
-		}
-	}
-
-	/**
-	 * Gets the `TrackerPosition` by its string designation.
-	 *
-	 * @return Returns an optional as not all strings are valid designators.
-	 */
-	public static Optional<TrackerPosition> getByDesignation(String designation) {
-		if (designation == null) {
-			return Optional.empty();
-		}
-
-		// Support old configs.
-		if (
-			designation.equalsIgnoreCase("body:left_leg")
-				|| designation.equalsIgnoreCase("body:left_knee")
-		) {
-			designation = "body:left_upper_leg";
-		} else if (
-			designation.equalsIgnoreCase("body:right_leg")
-				|| designation.equalsIgnoreCase("body:right_knee")
-		) {
-			designation = "body:right_upper_leg";
-		} else if (designation.equalsIgnoreCase("body:left_ankle")) {
-			designation = "body:left_lower_leg";
-		} else if (designation.equalsIgnoreCase("body:right_ankle")) {
-			designation = "body:right_lower_leg";
-		} else if (designation.equalsIgnoreCase("body:left_forearm")) {
-			designation = "body:left_lower_arm";
-		} else if (designation.equalsIgnoreCase("body:right_forearm")) {
-			designation = "body:right_lower_arm";
-		} else if (designation.equalsIgnoreCase("body:left_upperarm")) {
-			designation = "body:left_upper_arm";
-		} else if (designation.equalsIgnoreCase("body:right_upperarm")) {
-			designation = "body:right_upper_arm";
-		} else if (designation.equalsIgnoreCase("body:left_controller")) {
-			designation = "body:left_hand";
-		} else if (designation.equalsIgnoreCase("body:right_controller")) {
-			designation = "body:right_hand";
-		}
-
-		return Optional.ofNullable(byDesignation.get(designation.toLowerCase()));
-	}
```

Comment:
Doesn't removing this break some backwards compatability? I don't think this should be removed...

Reply:
backwards compatibility of what version of slimevr?

Reply:
Very old, do you think it's no longer useful? I guess it's not really worthwhile to support such old versions. Feel free to resolve this if you agree.

Reply:
There’s no reason to keep this much code for such old versions. Those specific trackers will just get unassigned is all. This code doesn’t even belong here in the first place; it should be in CurrentVRConfigConverter…